kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -38,7 +46,9 @@ impl KaniAttributeKind {
             | KaniAttributeKind::Solver
             | KaniAttributeKind::Stub
             | KaniAttributeKind::Unwind => true,
-            KaniAttributeKind::Unstable => false,
+            KaniAttributeKind::Unstable
+            | KaniAttributeKind::Ensures
+            | KaniAttributeKind::Requires => false,
```

Comment:
The default value is `false`? In CBMC, it's `true` (see https://diffblue.github.io/cbmc/contracts-requires-ensures.html).

Reply:
This is not the default value, this says whether this attribute can only occur on a harness (which is false for contracts).